exec/vector/src/main/java/org/apache/drill/exec/vector/accessor/convert/AbstractWriteConverter.java

```diff
@@ -143,4 +144,9 @@ public void setTime(LocalTime value) {
   public void setTimestamp(Instant value) {
     baseWriter.setTimestamp(value);
   }
+
+  @Override
+  public void copy(ColumnReader from) {
+    throw new UnsupportedOperationException("Cannot copy values through a type converter");
```

Comment:
If type converters shouldn't support copy then maybe makes sense to mark the method as final here.